server/fleet/software_installer.go

```diff
@@ -169,6 +169,62 @@ type HostSoftwareInstallerResult struct {
 	HostTeamID *uint `json:"-" db:"host_team_id"`
 	// UserID is the user ID that requested the software installation on that host.
 	UserID *uint `json:"-" db:"user_id"`
+	// InstallScriptExitCode is used internally to determine the output displayed to the user.
+	InstallScriptExitCode *int `json:"-" db:"install_script_exit_code"`
+	// PostInstallScriptExitCode is used internally to determine the output displayed to the user.
+	PostInstallScriptExitCode *int `json:"-" db:"post_install_script_exit_code"`
+}
+
+const (
+	SoftwareInstallerQueryFailCopy          = "Query didn't return result or failed\nInstall stopped"
+	SoftwareInstallerQuerySuccessCopy       = "Query returned result\nProceeding to install..."
+	SoftwareInstallerScriptsDisabledCopy    = "Installing software...\nError: Scripts are disabled for this host. To run scripts, deploy the fleetd agent with --scripts-enabled."
+	SoftwareInstallerInstallFailCopy        = "Installing software...\nFailed\n%s"
+	SoftwareInstallerInstallSuccessCopy     = "Installing software...\nSuccess\n%s"
+	SoftwareInstallerPostInstallSuccessCopy = "Running script...\nExit code: 0 (Success)\n%s"
+	// TODO(roberto): this is not true, how do we know that the rollback script was successful?
```

Comment:
We'll likely need another column in the DB to store along with the rest of the results. Next iteration, I'd say? Good enough for now to assume it tried its best? Or reflect the fact that we're not sure in the message ("attempted rollback").